File: smartdevices-app/smartdevicesapp.model/src/main/java/de/vogler_engineering/smartdevicesapp/model/entities/media/FileUploadResponse.java
/**
 * Copyright (c) Vogler Engineering GmbH. All rights reserved.
 * Licensed under the MIT License. See LICENSE.md in the project root for license information.
 */
package de.vogler_engineering.smartdevicesapp.model.entities.media;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.UUID;

import lombok.Data;

/**
 * Created by vh on 09.03.2018.
 */

@Data
public class FileUploadResponse {

    private boolean success;
    private UUID id;
    private long size;

    @JsonProperty("Content-Type")
    private String contentType;
}

File: smartdevices-gateway/SmartDevicesGateway.Api/Requests/FileUploadResponse.cs
//
// Copyright (c) Vogler Engineering GmbH. All rights reserved.
// Licensed under the MIT License. See LICENSE.md in the project root for license information.
//
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace SmartDevicesGateway.Api.Requests
{
    public class FileUploadResponse
    {
        public bool Success { get; set; }

        public Guid Id { get; set; }

        public long Size { get; set; }

        [JsonProperty("Content-Type", NullValueHandling = NullValueHandling.Ignore)]
        public string ContentType { get; set; }
    }
}
